mods/player_api/api.lua

```diff
@@ -208,6 +208,14 @@ minetest.register_globalstep(function()
 			end
 		end
 	end
+end
+
+-- Mods can modify the globalstep by overriding player_api.globalstep
+
+-- local for speed
+local player_api_globalstep = player_api.globalstep
```

Comment:
This nullifies the whole point of the PR, as the local variable will not get updated when `player_api.globalstep` changes.

Reply:
It works in the mod that needs this, and the following code produces a "no error" shutdown for me:

(`minetest.after(10,` so I can make sure it doesn't have my mod's animations)
```lua
minetest.after(10, function()
    assert(player_api.globalstep)
    function player_api.globalstep()
        error("No error")
    end

    minetest.after(5, error, "mod")
end)
```